app/jobs/send_event_slack_notifications_job.rb

```diff
@@ -0,0 +1,102 @@
+class SendEventSlackNotificationsJob < ApplicationJob
+  queue_as :default
+
```

Comment:
This job probably needs a `before_perform` to schedule the job occurring on the next hour interval.

A later PR could break this into two jobs: One to send notifications for each event and one to schedule a notification job for each event (so that it isn't restricted to on the hour). This would also need to keep track of events that have already been notified (and clear that record if the time changes to a later time).